### PR TITLE
Add timeouts for server

### PR DESCRIPTION
**Pull request by vivek378521**

### CODE REVIEW

**Comment by vivek378521** [comment]
Here is the test SS
Selection_016

File: constants/constants.go [hunk 1]
```diff
@@ -14,3 +14,6 @@ const S3_REGION = "us-east-1"
 const AWS_ENDPOINT = "http://localhost:4566"
 const PRESIGNED_URL_EXPIRATION = 60 * time.Minute
 const OUTPUT_FILE_PATH_PREFIX = "output"
+const READ_TIMEOUT = 1 * time.Second
+const WRITE_TIMEOUT = 1 * time.Second
```
Comment (uzaxirr) [inline constants/constants.go]: Set it to 5 Sec
i.e
```go
const READ_TIMEOUT = 5 * time.Second
const WRITE_TIMEOUT = 5 * time.Second
```

**Review by uzaxirr — CHANGES_REQUESTED** [review]

File: constants/constants.go (hunk 1 above)
Comment (vivek378521) [reply constants/constants.go]: Done.

**Review by uzaxirr — APPROVED** [review]
LGTM, Thank you for your contribution!

**Comment by abskrj** [comment]
we moved from gin to http server? @uzaxirr @vivek378521

**Comment by uzaxirr** [comment]
> LGTM, Thank you for your contribution!

What no, the microservice is still a Gin server.

**Comment by abskrj** [comment]
we are doing http.ServeAndListen() and gin.Run() both

**Pull request merged**